src/modules/dashboard/components/ActionsPage/TotalStakeWidget/TotalStakeWidget.tsx

```diff
@@ -0,0 +1,120 @@
+
+const displayName = 'TotalStakeWidget';
+
+export enum StakeSide {
+  Motion = 'MOTION',
+  Objection = 'OBJECTION',
+  Both = 'BOTH',
+}
+
+type Props = {
+  colonyAddress: Address;
+  motionId: number;
+  stakeSide?: string;
+};
+
+const TotalStakeWidget = ({
+  colonyAddress,
+  motionId,
+  stakeSide = StakeSide.Motion,
+}: Props) => {
+  const { walletAddress } = useLoggedInUser();
+  const { data, loading } = useStakeAmountsForMotionQuery({
+    variables: {
+      colonyAddress,
+      userAddress: walletAddress,
+      motionId,
+      stakeSide,
+    },
+  });
+  const { data: nativeTokenAddressData } = useColonyNativeTokenQuery({
+    variables: { address: colonyAddress },
+  });
+  const [fetchTokenInfo, { data: tokenInfoData }] = useTokenInfoLazyQuery();
+
+  useEffect(() => {
+    if (nativeTokenAddressData) {
+      const {
+        processedColony: { nativeTokenAddress },
+      } = nativeTokenAddressData;
+      fetchTokenInfo({ variables: { address: nativeTokenAddress } });
+    }
+  }, [fetchTokenInfo, nativeTokenAddressData]);
+
+  if (loading || !data?.stakeAmountsForMotion) {
+    return null;
+  }
+
+  const { totalStaked, userStake, requiredStake } = data.stakeAmountsForMotion;
+  const divisibleRequiredStake = !bigNumberify(requiredStake).isZero()
+    ? requiredStake
+    : 1;
+
+  const totalYAYStakedPercentage = bigNumberify(totalStaked.YAY || 0)
+    .mul(100)
+    .div(divisibleRequiredStake)
+    .toNumber();
+  const totalNAYStakedPercentage = bigNumberify(totalStaked.NAY || 0)
+    .mul(100)
+    .div(divisibleRequiredStake)
+    .toNumber();
+  const formattedTotalYAYStakedPercentage = formatNumber({
+    truncate: 2,
+  })(totalYAYStakedPercentage);
+  const formattedTotalNAYStakedPercentage = formatNumber({
+    truncate: 2,
+  })(totalNAYStakedPercentage);
+
+  return (
+    <div className={styles.widget}>
+      {stakeSide !== StakeSide.Both && (
+        <SingleTotalStake
+          userStake={userStake}
+          requiredStake={divisibleRequiredStake}
+          stakeSide={stakeSide}
+          totalPercentage={
+            stakeSide === StakeSide.Motion
+              ? totalYAYStakedPercentage
+              : totalNAYStakedPercentage
+          }
+          formattedTotalPercentage={
+            stakeSide === StakeSide.Motion
+              ? formattedTotalYAYStakedPercentage
+              : formattedTotalNAYStakedPercentage
+          }
+          tokenDecimals={tokenInfoData?.tokenInfo.decimals}
+          tokenSymbol={tokenInfoData?.tokenInfo.symbol}
+        />
+      )}
+      {stakeSide === StakeSide.Both && (
+        <GroupedTotalStake
+          requiredStake={requiredStake}
+          formattedTotalNAYStakedPercentage={formattedTotalNAYStakedPercentage}
+          formattedTotalYAYStakedPercentage={formattedTotalYAYStakedPercentage}
+          tokenDecimals={tokenInfoData?.tokenInfo.decimals}
+          tokenSymbol={tokenInfoData?.tokenInfo.symbol}
```

Comment:
Once you have stakes on both sides:
- you have two radio buttons
- a next button

you select which side you want to stake, you press the stake button, then it takes you to that side to stake (using the slider)

![Screenshot from 2021-04-26 10-58-08](https://user-images.githubusercontent.com/1193222/116049077-8c9b2500-a67e-11eb-9648-daab4c493b1b.png)

![Screenshot from 2021-04-26 11-00-15](https://user-images.githubusercontent.com/1193222/116049115-99b81400-a67e-11eb-8500-0e20c11428b3.png)


Your implementation is missing:
- the next button
- the logic to switch the user to either yay stake side or the nay stake side